lib/src/widgets/month_range_picker/month_view.dart

```diff
@@ -206,14 +212,14 @@ class _MonthViewState extends State<MonthView> {
                             int eMonth = calendarItem.indexWhere(
                                 (element) => element.month == endMonth());
                             widget.callback.call(sMonth + 1, eMonth + 1,
-                                startMonth(), endMonth(), currentYear);
+                                startMonth(), endMonth(), selectedYear);
                           }
                         : null,
                     child: Container(
                       margin: const EdgeInsets.symmetric(vertical: 12),
                       decoration: BoxDecoration(
                           color: e.isTapped
-                              ? Colors.blue.withOpacity(0.5)
+                              ? widget.selectedColour
```

Comment:
Don't make selected color required. If it's required the we use that otherwise we use default (Colors.blue.withOpacity(0.5)) as it was set earlier.